cmd/login/login.go

```diff
@@ -46,8 +53,31 @@ func NewLoginCmd(
 	return &cmd
 }
 
-func run() func(*cobra.Command, []string) error {
+func run(client login.Client) func(*cobra.Command, []string) error {
 	return func(cmd *cobra.Command, args []string) error {
+		deviceAuthorization, err := login.FetchDeviceAuthorization(
+			client,
+			login.ClientID,
+			login.GetDeviceName(),
+			viper.GetString(cliflags.BaseURIFlag),
+		)
+		if err != nil {
+			return errors.NewError(output.CmdOutputError(viper.GetString(cliflags.OutputFlag), err))
```

Comment:
looks like `errors.NewError(output.CmdOutputError(viper.GetString(cliflags.OutputFlag), err))` appears 11 times in the project -- might be a good candidate for a function

Reply:
Yep, sounds good.